### PR TITLE
Add Update Space

### PR DESCRIPTION
**Pull request by dlitvakb**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
 # Change Log
 
 ## Unreleased
+### Added
+* Adds `update_space` command to update already existing spaces using JSON Templates
+* Adds `--mark-processed` option to `create_space` and `update_space` to enforce marking which entries have already been processed
```
Comment (neonichu) [inline CHANGELOG.md]: > entries/assets

File: README.md [hunk 2]
```diff
@@ -81,11 +87,23 @@ Additionally, you can send an options hash with the following keys:
...
 }
 Contentful::Bootstrap::CommandRunner.new.create_space("space_name", options)
 ```
 
+To Update an existing Space
+
+```ruby
+options = {
+  json_template: "/path/to/template.json", # Will use the JSON file specified as a Template
+  mark_processed: false, # if true will mark all items as 'bootstrapProcessed and will be avoided on future update_space calls
```
Comment (neonichu) [inline README.md]: I think we used to say "resources" instead of "items" — don't think we have a consensus, though 😢

File: README.md [hunk 3]
```diff
@@ -147,6 +165,9 @@ Using the `--json-template` option, you can create spaces with your own predefin
 This can be useful for creating testing & development spaces or just starting new projects from
 a common baseline. You can find a complete example [here](./examples/templates/catalogue.json)
 
+Using the `--mark-processed` option alongside `--json-template` will mark all items as `bootstrapProcessed`,
+which will make it so `update_space` calls avoid already created items. (An item being either a Content Type, Entry or Asset).
```
Comment (neonichu) [inline README.md]: would also say "resources" here

**Comment by neonichu** [comment]
LGTM, except for minor `README` edits

File: README.md [hunk 4]
```diff
@@ -41,6 +41,12 @@ You can also generate JSON Templates from existing spaces by doing:
 $ contentful_bootstrap generate_json <space_id> <access_token> [--output-file OUTPUT PATH]
 ```
 
+You can update existing spaces from JSON Templates by doing:
+
+```bash
+$ contentful_bootstrap update_sapce <space_id> -j template_path [--mark-processed]
```
Comment (neonichu) [inline README.md]: > update_space

Comment (dlitvakb) [inline README.md]: What do you mean with this comment?

Comment (dlitvakb) [inline README.md]: figured it out finally 😛 fixing

**Comment by neonichu** [comment]
:shipit:

**Pull request merged**